### DISCUSSION TITLE
Public Registry Collection!!

### DISCUSSION THREAD

**Discussion by mathisloge** [microsoft/vcpkg#17161] [Show and tell]
Please post your public registry repositories into this discussion. 
This should provide an overview of the registries that already exists and help to avoid duplicate work.

Template:
```markdown
#### repository:
[your git url]
#### available ports:
[list the available ports (if the number of ports is not too big)]
#### Why a custom registry instead of a PR in vcpkg: 
Description why the ports in the registry aren't part of the built-in registry.
```

What is a registry? https://github.com/microsoft/vcpkg/blob/master/docs/specifications/registries-2.md

--- 
use the github tag `vcpkg-registry`:
https://github.com/topics/vcpkg-registry

**Comment by mathisloge** [comment]
#### repository:
https://github.com/mathisloge/vcpkg-registry
#### available ports:
lua51(version 5.1.3), lua52( version 5.2.4), lua53(version 5.3.6)
turf, junction
#### Why a custom registry instead of a PR in vcpkg: 
Because downgrading isn't allowed in vcpkg and as part of this policy this custom registry exists. See https://github.com/microsoft/vcpkg/pull/17135#issuecomment-815290153 and onwards for more details. 
For turf and junction see https://github.com/microsoft/vcpkg/pull/17540#issuecomment-829608222 and the previous comments for explanation

**Comment by luncliff** [comment]
Tag suggestion: https://github.com/topics/vcpkg-registry

**Reply by mathisloge** [reply]
added in the main discussion

**Comment by reitowo** [comment]
#### repository:
https://github.com/cnSchwarzer/vcpkg-registry
#### available ports:
[k1ee](https://github.com/cnSchwarzer/k1ee)
[bup](https://github.com/cnSchwarzer/BilibiliUp)
#### Why a custom registry instead of a PR in vcpkg: 
`k1ee` is a simple tools collection library, and its coding style is very personal, I don't think a PR is a good decision. 
`bup` seems only useful in China or some world video uploaders who use [bilibili.com](https://www.bilibili.com), so that it might not useful for most people.

**Comment by msclock** [comment]
#### repository:
https://github.com/msclock/cmake-registry
#### available ports:
```bash
├── cmake-conan
├── cmake-cpm
├── cmake-modules
├── cmake-stm32
├── rpavlik-cmake-modules
├── stablecoder-cmake-scripts
└── vcpkg-cmake-module
├── ...
```
#### Why a custom registry instead of a PR in vcpkg: 
What I want to do is to provide a place to share open source useful cmake modules but not cpp libraries.

**Comment by YLuc** [comment]
package cxxgraph needs update to 3.0.0
it stays on 2.0.0 for too long, and the new version has many changes

**Reply by Osyotr** [reply]
Instead of spamming you could've just do it yourself.
